library/core/src/iter/adapters/cloned.rs

```diff
@@ -0,0 +1,142 @@
+use crate::{
+    iter::{
+        adapters::zip::try_get_unchecked, adapters::TrustedRandomAccess, FusedIterator, TrustedLen,
+    },
+    ops::Try,
+};
```

Comment:
the style guide encourages using no multi-line importing.

Reply:
Hm, I couldn't find the style guide, can you link it? And why this is not enforced by `./x.py fmt`?

Reply:
I'm sorry, it's not a style guide but a prefer way to format code.
You could look around the codebase to find the general pattern.

About automatic enforcing, probably because it's hard to implement, rustfmt has [`imports_layout`](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#imports_layout) but it is nowhere close.

Reply:
Okay, I'll reformat the code...

This should become something like this?

```rust
use crate::ops::Try;
use crate::iter::adapters::{zip::try_get_unchecked, TrustedRandomAccess};
use crate::iter::{FusedIterator, TrustedLen};
```

Reply:
Yeah, the traits may group together.